src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
```

Comment:
The constructor of the Animal class should not accept a health parameter, as the health is always initialized to 100 by default according to the task description. You can remove the health parameter from the constructor and directly set `this.health = 100;` on line 6.